bin/oneoff/upload_non_en_videos.rb

```diff
@@ -0,0 +1,148 @@
+# This script is used to batch upload non-EN videos, with the assumption that
+# we will frequently get these in batches. They will be uploaded to YouTube
+# and S3 as well as added to videos.csv.
+
+require 'google/apis'
+require 'google/apis/youtube_v3'
+require 'google/api_client/client_secrets.rb'
+require 'googleauth'
+require 'googleauth/stores/file_token_store'
+require 'csv'
+require 'fileutils'
+require 'rubygems'
+require 'csv'
+require 'optparse'
+
+# Switch this to false when ready to run
+DRY_RUN = true
```

Comment:
Would it be easy to make this a command line option instead?

Reply:
I didn't make it that because I liked the idea of having to do something manually to flip the flag. But now that I think about it more, that could lead to this flag being accidentally committed as `false`. I'll change it.